### PR TITLE
Added the Using the Device Plug-in topic

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/fG3BrfFo/639-5psap-gpu-device-plugin-documentation

Preview Build:
http://file.rdu.redhat.com/~ahardin/03022018/gpu-device-plugin/scaling_performance/device_plugin.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@vikaschoudhary16 @jeremyeder  FYI.This is very much a WIP and this docs-driven development approach is pretty slow going. Still wading through the existing resources. I feel like I'm going to have a lot of questions for you soon. 

cc @zvonkok

**Comment by ahardin-rh** [comment]
@vikaschoudhary16 @jeremyeder Okay, here's a more complete draft after going through all of the information. The docs-driven development adventure continues! 🍻  Some questions:

1. I know a big use case is for Nvidia, but I assume this will also apply to other vendors. So, should we write this in more broad of terms, or with special Nvidia usage at this time? I ask because I can use replaceables in parts where we don't want to call out Nvidia as a specific vendor.

2. Is what I currently have accurate and does the user need to do all of the parts under setup? Did I take out too much? There was a lot of guesswork on my part, so your review is appreciated.

Thanks!
cc @zvonkok

**Comment by jeremyeder** [comment]
@sjennings @derekwaynecarr here's a start.

File: scaling_performance/device_plugin.adoc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+Your base system must be properly registered with Red Hat or have access to the
+proper set of `yum` repositories.
+
+[[device-plugin-build-the-container-images]]
+=== Build the Container Images
+
+. Build the parent image, `rhel7-cuda9`:
++
+----
+$ cd rhel7-cuda9
```
Comment (mdshuai) [inline scaling_performance/device_plugin.adoc]: Where is the rhel7-cuda9 ?

**Comment by mdshuai** [comment]
`DevicePlugins` default is false, we need doc how to enable it.

**Comment by vikaschoudhary16** [comment]
@ahardin-rh my PR, https://github.com/openshift/openshift-docs/pull/7630, could be a precursor to your PR . WDYT?

**Comment by ahardin-rh** [comment]
@vikaschoudhary16 That sounds good to me. Thanks for your PR! I will take a look.

File: scaling_performance/device_plugin.adoc [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[what-the-device-plugin-does]]
+== What the Device Plug-in Does
```
Comment (ahardin-rh) [inline scaling_performance/device_plugin.adoc]: This part will be replaced with https://github.com/openshift/openshift-docs/pull/7751/files#diff-a0529a66faa63bb82338644ecf45eec5

**Comment by ahardin-rh** [comment]
@vikaschoudhary16 @zvonkok Are we at a point where we can tighten up the technical details of this PR? Are we still targeting this for 3.9? This related PR on the conceptual information is ready for review: https://github.com/openshift/openshift-docs/pull/7751  PTAL. Thanks!

**Comment by ahardin-rh** [comment]
@zvonkok I updated this based on your latest notes. Thanks for all of your help!  Here's the preview build:
http://file.rdu.redhat.com/~ahardin/03022018/gpu-device-plugin/scaling_performance/device_plugin.html

Should something be said about why NVIDIA is used in all of the examples? Is this the only supported use case?

cc @jeremyeder @vikaschoudhary16

File: scaling_performance/device_plugin.adoc [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+=== Prerequisites
+* Your base system must be properly registered with Red Hat or have access to the
+proper set of `yum` repositories.
+
+* You must have {product-title} 3.9 xref:../install_config/index.adoc#install-config-install-index[installed].
+
+[[device-plugin-host-preparation]]
+=== Prepare the Host
+
+Before you can use GPUs in {product-title}, you must first prepare the host by
```
Comment (ahardin-rh) [inline scaling_performance/device_plugin.adoc]: @zvonkok What are the steps for this? What drivers are needed and why?

**Comment by ahardin-rh** [comment]
@zvonkok Updated with the latest details. Thanks!

File: scaling_performance/device_plugin.adoc (hunk 3 above)
Comment (ahardin-rh) [reply scaling_performance/device_plugin.adoc]: @zvonkok ^^

**Comment by ahardin-rh** [comment]
@zvonkok @vikaschoudhary16 Any updates needed?

**Comment by zvonkok** [comment]
@ahardin-rh  Not from my side, you pulled my latest changes. LGTM

**Comment by ahardin-rh** [comment]
@mdshuai Can you please review the latest changes?

**Pull request merged**